### PR TITLE
Ft jwt auth

### PR DESCRIPTION
**Pull request by Maxcutex**
**What does this PR do?**

This PR implements jwt and refactors admin get all vendors api to authenticate with jwt 

**Description of Task to be completed?**

• implement test to valid jwt authentication and one method in admin

*How should this be manually tested?*

• Run `export $(cat .env)`
• `pytest`
• Hit the endpoint

**Any background context you want to provide?**

All endpoints need to be authenticated before consumption

**What are the relevant pivotal tracker stories?**

None

### CODE REVIEW

File: app/views/user_manage_token.py [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+from app.helpers import validate_request
+
+user_menu_blueprint = Blueprint('user', 'user', url_prefix='/api/v1/')
+
+
+@user_menu_blueprint.route('/user/token', methods=['POST'])
+@validate_request((str, 'userEmail', 'userRole'))
+def post():
+    """
+    Creates a new menu
```
Comment (fob413) [inline app/views/user_manage_token.py]: I don't think this comment is meant to this view

**Comment by fob413** [comment]
The `@requires_auth_admin` decorator is only on 1 view out of 8 views that we currently have. Will that be done on this PR?

**Comment by Maxcutex** [comment]
made the changes for comment and .env_sample. I actually did 1 so that you can inmplement the rest going forward.

**Comment by Maxcutex** [comment]
One more thing to review before i merge. The reissue of jwt

**Comment by Maxcutex** [comment]
last review then i merge

**Pull request merged**